flab_bada/domain/email/email_service.py

```diff
@@ -67,3 +84,13 @@ def verify_secret_num(self, email: str, secret_key: str) -> bool:
         if secret_key == mem_secret_key:
             check_data = True
         return check_data
+
+    def verify_confirm_token(self, email: str, token: str) -> bool:
+        """이메일 인증 토큰 검사"""
+        check_data = False
+
+        # 메모리디비에서 값을 얻는다.
+        token_key = self.get_secret_num(email)
+        if token == token_key:
+            check_data = True
+        return check_data
```

Comment:
```suggestion
        return token == self.get_secret_num(email)
```
동일한 의미를 갖도록 코드를 간결하게 작성하는 연습도 해보세요 
코드만 읽어도 충분히 그 의미가 해석되면 주석을 굳이 달아야할 필요가 없어집니다

Reply:
아 감사합니다. ~ 위에는 진짜 버릇 처럼 저렇게 하는 듯 합니다. 
간결하게 작성하는 방법 연습 해보겠습니다.